test/local_integration_test.py

```diff
@@ -113,6 +120,15 @@ def check_bdbag_endpoint(self, lambda_endpoint: str, url: str):
                     self.assertTrue(num_rows > 1)
                     logger.info(f'BDBag file {file_name} contains {num_rows} rows.')
 
+    def download_file_from_drs_response(self, response):
+        """Assert HTTP status code of file download is 200."""
+        json_data = response.json()['data_object']
+        url = first(json_data['urls'])['url']
+        file_name = json_data['name']
+        r = self.requests.get(url)
+        r.raise_for_status()
+        logger.info(f'Verified file {file_name} can be downloaded.')
```

Comment:
I'd need some evidence that the file is actually a fastq.

Reply:
Inserted two assertions, one for the file name in data object from the JSON response, and another on the file name of the file written to disk (to a temporary directory). IMHO, opening the file and checking its contents is beyond the scope of our responsibility.

Reply:
^^^obsolete - now opening the files and checking characteristic bytes that indicate `.gz` and `.fastq`, respectively.